### DISCUSSION TITLE
Separate Enchantment Table and interaction permission

### DISCUSSION THREAD

**Discussion by nivcoo** [BG-Software-LLC/SuperiorSkyblock2#484] [Pending Feature Requests] [answered]
Hello, can you add condition to separate the enchantment table interaction withslab interaction ? I know it's the same event but you can add condtion and separate that. Thx

**Comment by Lemvs** [comment, marked as answer]
Implemented some time ago with new interactables.yml format.

@OmerBenGera you can mark this answered and close it.
